core/src/main/java/org/springframework/security/core/context/ThreadLocalSecurityContextHolderStrategy.java

```diff
@@ -45,6 +45,11 @@ public SecurityContext getContext() {
 		return ctx;
 	}
 
+	@Override
```

Comment:
Will you please update the copyright message for each edited file to include the year `2021`?